docs/Contributing/Issues/issues.md

```diff
@@ -0,0 +1,67 @@
+## Using the issue tracker
+
+The [issue tracker](https://github.com/RI-SE/ATOS/issues) on the repo page is the preferred channel for [bug reports](#bugs),
+features requests and for submitting pull
```

Comment:
Can you submit pull requests via the issues tab?

Reply:
From an issue, you can create a branch which wiich will be linked to the issue.
![image](https://github.com/RI-SE/ATOS/assets/10436623/f24b3c92-d740-456c-8045-29d1c508242c)

![image](https://github.com/RI-SE/ATOS/assets/10436623/8e0541fd-18f5-4bf2-b787-18becdce5b6d)

This can then after proper review etc become a pr that is easy to correlate with the issue.

Reply:
The sentence in the text still has to be modified for clarity.
Can you use the issue page in the repo for submitting pull requests? 

**The issue tracker on the repo page is the preferred channel** for [bug reports](#bugs),
features requests and for **submitting pull requests**

Reply:
Modified this and added an image for more clarity